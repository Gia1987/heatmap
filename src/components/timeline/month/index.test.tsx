import { render } from "@testing-library/react";
import Month from "./";
import dayjs from "dayjs";

const Component = (props = { startDate: dayjs("2019-01-01"), index: 5 }) => {
  return <Month {...props} />;
};

describe("<Month />", () => {
  it("renders Feb", () => {
    const { container } = render(Component());

    expect(container).toMatchSnapshot();
  });
});
