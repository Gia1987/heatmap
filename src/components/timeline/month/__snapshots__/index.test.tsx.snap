// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Month /> renders Feb 1`] = `
<div>
  <div
    class="timeline-months-month Feb"
  >
    Feb
  </div>
</div>
`;
